--- src/logger.go.orig	2026-03-04 18:16:33.019587059 -0700
+++ src/logger.go	2026-03-04 18:27:20.690760299 -0700
@@ -126,6 +126,8 @@
 		// Both v4 and flip have the same resolution
 		// We'll label them as v4 by default, could be either
 		globalApp.DeviceModel = "miyoo-mini-v4"
+	} else if w == 480 && h == 640 {
+		globalApp.DeviceModel = "miyoo-a30"
 	} else {
 		globalApp.DeviceModel = fmt.Sprintf("unknown-%dx%d", w, h)
 	}
